### DISCUSSION TITLE
KICS security standard benchmark

### DISCUSSION THREAD

**Discussion by ghost** [Checkmarx/kics#6054] [Q&A] [answered]
Hi team,

I am investigating KICS for our projects. and we would like to know what security benchmarks that KICS queries are based on. Is there any information or links that I can refer to? any link to future plan would also help.  Thanks

**Comment by gabriel-cx** [comment, marked as answer]
Hi @MayaShenFraedom ,

In our website we have a [page](https://docs.kics.io/latest/certifications/) that can give you information regarding security benchmarks related to KICS queries. Hope this information helps you with what you need!
